### ISSUE TITLE
Tiktok support

### ISSUE THREAD

**Issue by MayhemBill** [Cog-Creators/Red-DiscordBot#5933]
### What component of Red (cog, command, API) would you like to see improvements on?

Streams

### Describe the enhancement you're suggesting.

I would like support for TikTok videos/streams like twitch go lives/ YouTube video posts.

**Comment by Drapersniper** [comment]
Not something that is or shouldn't be done by Red, Lavalink should support it - in fact there is a plug-in for lavalink that enables TikTok - once plug-in support is added red and if the decision to support an unofficial plug-in is made or native TikTok support is added to lavaplayer then and only then would a PR to unblock TikTok videos should be done to Audio/RLL

If you are talking about adding TikTok support to the Audio Cog

**Comment by MayhemBill** [comment]
No I am not, I mean like how the streams cog sends a notification when a video is uploaded to youtube or a user on twitch goes live, I want a notification when a video is posted on tiktok

**Comment by yamikaitou** [comment]
Streams only handles live streams, not video uploads

**Comment by MayhemBill** [comment]
Oh, maybe I misunderstood what the youtube key was for then?

**Comment by Zephyrkul** [comment]
Youtube is also a livestreaming platform, so Streams supports sending livestream announcements. Streams does *not* post when a youtube video is uploaded, all it is for is livestreams.

**Comment by Flame442** [comment]
Seems like this is out of scope for that cog, so I'm closing this issue now. While something like what you are looking for may be possible through a third party cog, it probably isn't the best fit for core.
